### DISCUSSION TITLE
Impossible to install created formula

### DISCUSSION THREAD

**Discussion by riffard** [Homebrew/discussions#286] [Everyday usage] [answered]
Hello, I created a new formula using

This message is a duplicate of the one posted here: https://discourse.brew.sh/t/impossible-to-install-a-created-formula/9289
I am posting it here too as the previous service will be deprecated soon.

brew create https://gitlab.cern.ch/gaudi/Gaudi/-/archive/v35r0/Gaudi-v35r0.tar.gz  --set-name gaudi
I tried to install it right after and it worked. After some modification to the formula, I obtain the following message when I tried to install it using `brew install gaudi`
```
==> Searching for similarly named formulae...
This similarly named formula was found:
gaudi
To install it, run:
  brew install gaudi
Error: No available formula or cask with the name "gaudi".
==> Searching taps on GitHub...
Error: No formulae found in taps.
```
I also tried to run the audit using `brew audit --new gaudi`:
```
Error: No available formula or cask with the name "gaudi".
```

Any idea why this is happening ?
Thanks,
Quentin

**Comment by Rylan12** [comment, marked as answer]
I think this is due to an issue with the formula code which is causing Homebrew to not be able to find the formula. Try running `brew readall` and see if there are any errors. If not, maybe post the formula code?

**Reply by riffard** [reply]
That helped me to debug it, thank you ! I had a double double quotes to close a string...
